pkg/reconciler/kubernetes/adapter.go

```diff
@@ -30,13 +42,20 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: ""`
+	progressTrackerInterval   = 5 * time.Second
+	progressTrackerTimeout    = 2 * time.Minute
```

Comment:
We have both `progressTrackerInterval` and `progressTrackerTimeout` defined [here](https://github.com/kyma-incubator/reconciler/blob/main/pkg/reconciler/instances/eventing/preaction/common.go#L8). 
Does it make sense to align those values?

Reply:
The value in common.go will be used when we call
`NewProgressTracker(client kubernetes.Interface, logger *zap.SugaredLogger, config Config)` to overwrite the config value.
But this `progressTrackerInterval` and `progressTrackerTimeout` here are used to create
` NewKubernetesClient(kubeconfig string, logger *zap.SugaredLogger, config *Config)`, which used by internal function `newProgressTracker()` to receive the value defined by it, although the value is the same, I think it's fine they are defined differently.
```
func (g *kubeClientAdapter) newProgressTracker() (*progress.Tracker, error) {
	clientSet, err := g.Clientset()
	if err != nil {
		return nil, err
	}
	return progress.NewProgressTracker(clientSet, g.logger, progress.Config{
		Interval: g.config.ProgressInterval,
		Timeout:  g.config.ProgressTimeout,
	})
}

```